turbinia/lib/utils.py

```diff
@@ -112,6 +112,10 @@ def extract_files(file_name, disk_path, output_dir, credentials=[]):
   Raises:
     TurbiniaException: If an error occurs when running image_export.
   """
+  if not disk_path:
+    raise TurbiniaException(
+        'image_export.py failed: Attempted to run with no local_path')
```

Comment:
Do you happen to know under what conditions this happens?  Is that from a particular evidence type?  I think it's likely a bug in the pre-processors if this happens since that's where the `local_path` gets set.  I think this change is fine to put in, but I would also like to fix the underlying problem.

Reply:
It could also happen if a Task didn't have the right `REQUIRED_STATES` set.

Reply:
I'm not entirely sure where the underlying bug is, but it occured when I was processing a disk image that had 55 Docker containers inside it